### ISSUE TITLE
Use local data for Company facts

### ISSUE THREAD

**Issue by redpoint13** [dgunning/edgartools#56]
Can the bulk data be downloaded from the SEC (https://www.sec.gov/edgar/sec-api-documentation)? For a Company, the data would be accessed locally from the companyfacts file (https://www.sec.gov/Archives/edgar/daily-index/xbrl/companyfacts.zip), with the hope of speeding up large data downloads, like when accessing all filings from the SP500.

**Comment by dgunning** [comment]
Hey, thanks for the suggestion, it's worth looking into. I'll do a quick scan this weekend then a deeper dive later.

**Comment by dgunning** [comment]
Testing local storage now

**Comment by dgunning** [comment]
Released 2.25.0 with bulk data downloaded to local storage for facts and submissions.
```
from edgar import *

use_local_storage() # OR os.environ['EDGAR_USE_LOCAL_DATA'] = "1"
download_edgar_data()
```
Should work as normal but faster. I think the local submissions limited to latest 1000 filings

**Comment by dgunning** [comment]
Closed as fixed
